contrib/opencensus-ext-grpc/tests/test_server_interceptor.py

```diff
@@ -110,7 +133,9 @@ def test_intercept_handler_exception(self):
                 None, None)
             mock_context = mock.Mock()
             mock_context.invocation_metadata = mock.Mock(return_value=None)
-            mock_context._rpc_event.call_details.method = 'hello'
+
+            mock_context._rpc_event.call_details.host = MOCK_HOST
+            mock_context._rpc_event.call_details.method = MOCK_METHOD
```

Comment:
Update tests to include the additional fields that were added?

Reply:
It's already included in the latest test_server_interceptor file.

![image](https://user-images.githubusercontent.com/6319967/97845000-44610280-1cec-11eb-89ea-6cbbe7d8f0ce.png)